itou/templates/apply/includes/list_card_body_siae.html

```diff
@@ -0,0 +1,136 @@
+{% load str_filters %}
+
+<div class="card-body">
+    <div class="fs-sm d-flex align-items-center mb-3">
+        <span class="text-secondary flex-grow-1">
+            Émise le {{ job_application.created_at|date:"d F Y" }} par
+ 
+            {% if request.user == job_application.sender %}
+                <strong>Vous</strong>
+            {% elif job_application.sender_kind == SenderKind.JOB_SEEKER %}
+                <strong>Le candidat lui-même</strong>
+            {% elif job_application.sender_kind == SenderKind.SIAE_STAFF %}
+                <strong class="mr-2">{{ job_application.sender_siae.display_name }}</strong>
+                {% if request.user.is_siae_staff %}
+                    <span class="badge badge-pill badge-sm badge-info">
+                        {% if job_application.origin == JobApplicationOrigin.PE_APPROVAL %}
+                            <i class="ri-hotel-line"></i>&nbsp;Employeur (Import agrément Pôle emploi)
+                        {% else %}
+                            <i class="ri-hotel-line"></i>&nbsp;Employeur (Auto-prescription)
+                        {% endif %}
+                    </span>
+                {% endif %}
+            {% elif job_application.sender_kind == SenderKind.PRESCRIBER %}
+                <strong class="mr-2">{{ job_application.sender_prescriber_organization.display_name }}</strong>
+                {% if job_application.sender_prescriber_organization %}
+                    {% if job_application.is_sent_by_authorized_prescriber %}
+                        <i class="ri-group-line"></i>&nbsp;Prescripteur habilité
+                    {% else %}
+                        <i class="ri-group-line"></i>&nbsp;Orienteur
+                    {% endif %}
+                {% endif %}
+            {% endif %}
+
+        </span>
+        {% include "apply/includes/state_badge.html" with job_application=job_application %}
+    </div>
+    {% with all_jobs=job_application.selected_jobs.all %}
```

Comment:
J'ai modifié la candidature de l'EI Job Insersion 90 pour que tu vois ce que ça donne quand il y a plusieurs métiers.
Cela ne correspond pas à la maquette.
![image](https://user-images.githubusercontent.com/7632730/227490908-cb72f4a9-b84d-48dc-a14f-23a20c6661ee.png)
Il faudrait avoir : 
![image](https://user-images.githubusercontent.com/7632730/227492100-bdbf830d-f9a7-4ab1-97e8-cc35291b8152.png)

Je ne sais pas si cette différence est voulue ou non par les designers.

1) Dans le cas où l'on reste sur l'affichage que tu as mis (et qui est celui du composant utilisé dans l'espace salarié) je te propose de faire un template commun pour ici et https://github.com/betagouv/itou/blob/master/itou/templates/approvals/includes/job_applications.html#L11-L57
tu pourras l'importer aux deux endroits avec `{% include "path/to/template.html" %}`
Comme ça on utilise le même code partout.
Il y a du mieux dans ta proposition (le slice par exemple) donc ça vaut le coup de faire un truc un peu entre les deux.
Il faut par contre que tu sortes la partie candidature spontatée comme je l'ai fait dans l'autre composant, ça rend la lecture beaucoup plus simple.

2) Dans le cas où le métier décide qu'ils veulent un affichage différent, je propose de ré-organiser le code de la manière suivante : 
```
{% if all_jobs|length = 0 %}
  <span class="font-weight-bold mr-1 mr-md-2">Candidature spontanée</span>
{% elif all_jobs|length = 1 %}
  {% with all_jobs|first as first_job %}
    <i class="ri-briefcase-line ri-xl mr-1"></i>
    <span class="font-weight-bold mr-1 mr-md-2">{{ first_job.display_name }}</span>
    {% if first_job.location %}
      <span class="fs-sm text-nowrap">
        <i class="ri-map-pin-2-line ri-sm mr-1"></i>
        {{ first_job.location }}
      </span>
    {% endif %}
  {% endwith %}
{% else %}
  <a class="d-flex flex-wrap align-items-center text-decoration-none has-collapse-caret collapsed"
                   data-toggle="collapse"
                   href="#collapseJobs"
                   role="button"
                   aria-expanded="false"
                   aria-controls="collapseExample">
    Voir les {{ other_job_number }} postes
  </a>
  <ul>
    {% for job in all_jobs%}
    ...
    {% endfor %}
  </ul>
{% endif %}
```

3) La dernière possibilité est qu'ils décident de mettre aussi cette nouvelle manière d'afficher les postes sur le PASS IAE, auquel cas, tu pourras quand même faire un template commum comme en 1), mais avec le code de 2)

Reply:
En fait ça dépend de la maquette… quelle version est la bonne ?
![Screenshot from 2023-03-27 09-35-23](https://user-images.githubusercontent.com/154904/227872656-1df5cfa0-d1ed-4892-90e8-8f3837b0ccfd.png)